python/_brotli.cc

```diff
@@ -88,6 +87,38 @@ static int lgblock_convertor(PyObject *o, int *lgblock) {
   return 1;
 }
 
+static BROTLI_BOOL compress_stream(BrotliEncoderState* enc, BrotliEncoderOperation op,
+                                   std::vector<uint8_t>* output, uint8_t* input, size_t input_length) {
+  BROTLI_BOOL ok = BROTLI_TRUE;
+
+  size_t available_in = input_length;
+  const uint8_t* next_in = input;
+  size_t available_out = 0;
+  uint8_t* next_out = NULL;
+
+  while (ok) {
+    ok = BrotliEncoderCompressStream(enc, op,
```

Comment:
@eustas The `available_in` and `next_in` variables are now outside of the loop.  In my testing, if `available_out` is `0`, then it won't be modified by `BrotliEncoderCompressStream`.  Would you prefer that I reset `available_out` to `0` each call?